source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -0,0 +1,179 @@
+.. include:: <isonum.txt>
+
+Robot Battery
+=============
+
+The power supply for an FRC\ |reg| robot is a single 12V 18Ah SLA (Sealed Lead Acid) AGM (Absorbent Glass Mat) battery, capable of high momentary currents well in excess of 180A when fully charged. The Robot Battery assembly includes the COTS battery, lead cables with contacts, and SB connector.
+
+Battery Safety & Handling
+-------------------------
+
+A healthy battery is **ALWAYS** "On" and the terminals are **always** energized. If the polarities short together - for example, a wrench or aerosol can falls and bridges the gap between two bare terminals - they will release all their stored energy in a big, dangerous arc. This risk drives a wide range of best practices: covering terminals in storage, only uncovering and working on one terminal or polarity at a time, only using Battery Leads as complete assemblies with SB contacts fully inserted, and covering the SB connector contacts with "battery tabs" once the Robot Battery is fully charged, among others.
```

Comment:
```suggestion
A healthy battery is **always** "On" and the terminals are **always** energized. If the polarities short together - for example, a wrench or aerosol can falls and bridge the gap between two bare terminals, causing all stored energy to be released in a dangerous arc. This risk drives a wide range of best practices: covering terminals in storage, only uncovering and working on one terminal or polarity at a time, only using Battery Leads as complete assemblies with SB contacts fully inserted, and covering the SB connector contacts with "battery tabs" once the Robot Battery is fully charged, among others.
```

Caps are a bit excessive and overly aggressive. The bold is good enough. Additionally, I reworded a sentence.

This paragraph mentions SB, but it's never explained what SB is